plugins/devtools/src/components/Content/InfoContent/InfoContent.tsx

```diff
@@ -72,7 +73,17 @@ const copyToClipboard = ({ about }: { about: DevToolsInfo | undefined }) => {
 /** @public */
 export const InfoContent = () => {
   const classes = useStyles();
+  const [resources, setResources] = useState<string | undefined>(undefined);
   const { about, loading, error } = useInfo();
+  useSignalApi('devtools:resources', message => {
```

Comment:
Naming convention here would be to just call this `useSignal`. As with other cases I do wonder if there's some structure that we could use here to make it easier to work with signals of different shapes, a `SignalRef` kinda thing exported from a common package, used both when sending the signal in the backend and receiving it in the frontend. Always nice to keep things a bit type safe.

Either way please don't experiment with that in this PR, let's git it shipped first 😁